docs/docsite/rst/user_guide/playbooks_reuse.rst

```diff
@@ -1,72 +1,175 @@
 .. _playbooks_reuse:
 
-Creating Reusable Playbooks
-===========================
+**************************
+Re-using Ansible artifacts
+**************************
 
-.. toctree::
-   :maxdepth: 1
+You can write a simple playbook in one very large file, and most users learn the one-file approach first. However, breaking tasks up into different files is an excellent way to organize complex sets of tasks and reuse them. Smaller, more distributed artifacts let you re-use the same variables, tasks, and plays in multiple playbooks to address different use cases. You can use distributed artifacts across multiple parent playbooks or even multiple times within one playbook. For example, you may want to update your customer database as part of several different playbooks. A task file lets you re-use the tasks that update the database in many playbooks while only maintaining them in one place.
 
-   playbooks_reuse_includes
-   playbooks_reuse_roles
+.. contents::
+   :local:
 
-While it is possible to write a playbook in one very large file (and you might start out learning playbooks this way), eventually you'll want to reuse files and start to organize things. In Ansible, there are three ways to do this: includes, imports, and roles.
+Creating re-usable files and roles
+==================================
 
-Includes and imports (added in Ansible version 2.4) allow users to break up large playbooks into smaller files, which can be used across multiple parent playbooks or even multiple times within the same Playbook.
+Ansible offers four distributed, re-usable artifacts: variables files, task files, playbooks, and roles.
 
-Roles allow more than just tasks to be packaged together and can include variables, handlers, or even modules and other plugins. Unlike includes and imports, roles can also be uploaded and shared via Ansible Galaxy.
+  - A variables file contains only variables.
+  - A task file contains only tasks.
+  - A playbook contains at least one play, and may contain variables, tasks, and other content. Tightly focused playbooks may work for re-use, but they cannot be re-used dynamically.
```

Comment:
s/may/can or s/may/might depending on what you're aiming to say

may = permission
can = ability/potential
might = possibility

I'd go with "can" because it mirrors the "cannot" used later.